taskmgr/task_status_updater.go

```diff
@@ -36,7 +36,7 @@ const (
 )
 
 type updateError struct {
-	StatusCode int
+	statusCode int
```

Comment:
Would it be useful to make these exported fields, so that if a caller of `UpdateTaskStatus` receives an error, they can inspect the http status code that caused the error? I believe at the moment they can get the message, and parse it to determine the status code, but maybe it is useful to expose it?